### PR TITLE
Tweak: Added action links to the WordPress Customizer [ED-13161]

### PR DESCRIPTION
**Pull request by rami-elementor**
This code was originally removed in Hello Theme 2.9.0, as we wanted to move all the settings to the new “Hello Theme Settings” screen inside the WordPress dashboard.

In Elementor 3.0.0 we want to revert this change, and bring back the action links to the WordPress Customizer screen.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -217,6 +217,20 @@ function hello_elementor_add_description_meta_tag() {
 // Header & footer styling option, inside Elementor
 require get_template_directory() . '/includes/elementor-functions.php';
 
+// Customizer controls
+function hello_elementor_customizer() {
```
Comment (bainternet) [inline functions.php:221]: not wrapping with function_exists ??

File: includes/customizer-functions.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		'hello-header-footer',
+		[
+			'sanitize_callback' => false,
+			'transport' => 'refresh',
+		]
+	);
+
+	$wp_customize->add_control(
+		new HelloElementor\Includes\Customizer\Hello_Customizer_Action_Links(
+			$wp_customize,
```
Comment (bainternet) [inline includes/customizer-functions.php:33]: are we sure it exsists? and loaded?

**Review by bainternet — COMMENTED** [review]
see comments

File: includes/customizer-functions.php (hunk 2 above)
Comment (rami-elementor) [reply includes/customizer-functions.php:33]: We get the `$wp_customize` parameter from the [customize_register](https://developer.wordpress.org/reference/hooks/customize_register/) hook.

WordPress best practices doesn't require us to check it when hooking to the customizer registration.

**Review by arielk — APPROVED** [review]

**Pull request merged**